materialize-rockset/README.md

```diff
@@ -0,0 +1,105 @@
+# Materialize Rockset
+
+This is a Flow connector that materializes delta updates of each document into a Rockset collection.
+
+## Getting Started
+
+Connector images are available at `ghcr.io/estuary/materialize-rockset`.
+
+The connector configuration must specify a [Rockset API key](https://rockset.com/docs/iam/#api-keys), which can be created in the [Rockset console](https://console.rockset.com/apikeys).
+
+For each Flow collection you'd like to materialize, add a binding with the names of the target Rockset workspace and collection. Both the workspace and collection will be created automatically by the connector if they don't already exist.
+
+**Example flow.yaml:**
+
+```yaml
+materializations:
+  example/toRockset:
+    endpoint:
+      flowSink:
+        image: ghcr.io/estuary/materialize-rockset:dev
+        config:
+          api_key: <your rockset API key here>
+          max_concurrent_requests: 5
+    bindings:
+      - resource:
+          workspace: <your rockset workspace name>
+          collection: <your rockset collection name>
+        source: example/flow/collection
+```
+
+## Bulk ingestion for large backfills of historical data
```

Comment:
It might be worth adding a simple flowchart or something to help describe the state machine that's happening. It's a workflow that requires user intervention at a few steps and could be easy to mess up.

Reply:
Yeah, that's a good idea. I think perhaps the user-facing docs might be a better place for it, though, since that would allow using mermaid. @oliviamiannone does that seem reasonable to you? I can help with writing the mermaid if you want.

Reply:
That sounds good to me, I'm not familiar with mermaid so a bit of help would probably save substantial time on my end! Also you might have a better idea of what it should look like. Though I'm happy to contribute as much as I can.